packages/ui-components/src/components/form/RadioGroup.vue

```diff
@@ -71,20 +79,22 @@
 </template>
 
 <script setup lang="ts" generic="Value extends string">
-import { CheckIcon, InformationCircleIcon } from '@heroicons/vue/24/outline'
+import { InformationCircleIcon } from '@heroicons/vue/24/outline'
 import type { ConcreteComponent } from 'vue'
 
 type OptionType = {
   value: Value
   title: string
+  subtitle?: string
   introduction?: string
-  icon: ConcreteComponent
+  icon?: ConcreteComponent
   help?: string
 }
 
 defineProps<{
   options: OptionType[]
   disabled?: boolean
+  stackOptions?: boolean
```

Comment:
Nitpick: `stackOptions` sounds more like an enum type value, maybe `isStacked` is easier to understand